vmm/sandbox/src/qemu/config.rs

```diff
@@ -21,7 +21,7 @@ use std::{
 };
 
 use containerd_sandbox::error::{Error, Result};
-#[cfg(target_arch = "x86_64")]
+#[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
```

Comment:
Is this cfg not needed anymore after aarch64 is supportted? If so, you can remove it.

Reply:
done